docs/flow-visibility.md

```diff
@@ -0,0 +1,85 @@
+# Flow Visibility
+## Purpose
+Antrea supports sending IPFIX flow records through a flow exporter. The Elastic
+Stack (ELK Stack) works as the data collector, data storage and visualization tool
+for flow records and flow-related information. This doc provides guidelines for
+deploying Elastic Stack with support for Antrea-specific fields in a Kubenetes
+cluster.
+
+## About Elastic Stack
+[Elastic Stack](https://www.elastic.co) is a group of open source products to
+help collect, store, search, analyze and visualize data in real time. We will
+use Logstash, Elasticsearch and Kibana in Antrea flow visualization.
+[Logstash](https://www.elastic.co/logstash) works as data collector to
+centralize flow records. [Logstash Netflow codec plugin](https://www.elastic.co/guide/en/logstash/current/plugins-codecs-netflow.html)
+supports Netflow v5/v9/v10(IPFIX) protocols for flow data collection.
+Flow exporter feature in Antrea Agent uses IPFIX (Netflow v10) protocol to
+export flow records.
+
+IPFIX flow records contain following Antrea specific fields along with standard
```

Comment:
Exported IPFIX flow records contain the following Antrea-specific fields along with ...